cluster-autoscaler/cloudprovider/ionoscloud/mock_ionos_cloud_manager_test.go

```diff
@@ -14,14 +14,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Code generated by mockery v2.3.0. DO NOT EDIT.
+// Code generated by mockery 2.11.0. DO NOT EDIT.
```

Comment:
Heavily outdated. ;)
But I'd actually ask for a go-generate-marker creating both mock files. Right now there seems to be no well-defined way to generate them.

Reply:
I'm hesitant to add `go:generate` markers. It's kind of a global thing. I can add a line to the README or something